.github/workflows/positron-python-nightly.yml

```
@@ -107,3 +118,11 @@ jobs:
       - name: Run Positron IPyKernel unit tests
         run: pytest python_files/positron
```

```suggestion
        run: pytest python_files/positron --junit-xml=./extensions/positron-python #wherever you want the path to be
```

if you need to, it looks like you can set a [flag in this call to generate the xml file](https://docs.pytest.org/en/stable/how-to/output.html#creating-junitxml-format-files)